### PR TITLE
Add docs for merging OpenAPI into a single file

### PR DESCRIPTION
**Pull request by alee792**
Adding a quick section to OpenAPI docs t describe generation into a single, merged file.

I also fixed the hierarchy of the templating section. I also added another "enums_as_ints" section so that "merge" wouldn't get lonely. It serves as a reference for future additions.

No issue, requested in gophers slack:
https://gophers.slack.com/archives/CBATURP1D/p1632504587076100

### CODE REVIEW

**Comment by alee792** [comment]
@googlebot I signed it!

**Comment by johanbrandhorst** [comment]
Hi, it looks like the CLA bot is a bit confused about your commit - it needs to be able to link your commit with your github account. Could you try recreating the commit with the same email you use for github?

File: docs/docs/mapping/customizing_openapi_output.md [hunk 1]
```diff
@@ -186,20 +185,66 @@ The content of `tables.md`:
...
 ![Screenshot OpenAPI file in Postman](../../assets/images/gotemplates/postman.png)
 
 For a more detailed example of a proto file that has Go, templates enabled, [see the examples](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/internal/proto/examplepb/use_go_template.proto).
 
+## Other plugin options
+
+A comprehensive list of OpenAPI plugin options can be found [here](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/protoc-gen-openapiv2/main.go). Options can be passed via `protoc` CLI:
+
+```sh
+--openapiv2_out . bar=baz,color=red:.
```
Comment (johanbrandhorst) [inline docs/docs/mapping/customizing_openapi_output.md:209]: This isn't correct use of protoc, I think you want this
```suggestion
--openapiv2_out . --openapiv2_opt bar=baz,color=red
```

File: docs/docs/mapping/customizing_openapi_output.md [hunk 2]
```diff
@@ -186,20 +185,66 @@ The content of `tables.md`:
...
+```
+
+Or, with `buf` in `buf.gen.yml`:
```
Comment (johanbrandhorst) [inline docs/docs/mapping/customizing_openapi_output.md:212]: Nit: we use `buf.gen.yaml` generally, though we support both.
```suggestion
Or, with `buf` in `buf.gen.yaml`:
```

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
Thanks a lot for this!

**Comment by alee792** [comment]
@googlebot I consent.

**Comment by alee792** [comment]
Alright, I bungled git here. Let's try this again 😅

File: go.mod [hunk 3]
```diff
@@ -11,6 +11,6 @@ require (
 	github.com/rogpeppe/fastuuid v1.2.0
 	golang.org/x/oauth2 v0.0.0-20210819190943-2bc19b11175f
 	google.golang.org/genproto v0.0.0-20210927142257-433400c27d05
-	google.golang.org/grpc v1.41.0
+  google.golang.org/grpc v1.41.0
```
Comment (johanbrandhorst) [inline go.mod:14]: Not sure what happened here, but this is the last thing that needs fixing here.

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
Lets just fix the mod file changes, then this looks good to go!

**Review by johanbrandhorst — APPROVED** [review]

**Comment by johanbrandhorst** [comment]
Thanks for your contribution 😍

**Pull request merged**